services/mailer.js

```diff
@@ -1,13 +1,16 @@
 const mailgun = require("mailgun-js");
 const logger = require("../lib/log")(__filename);
-const getEnvVar = require("../lib/getEnvVar");
+require("dotenv").config();
 
-const mg = mailgun(getEnvVar("mailgun"));
+const mg = mailgun({
+  apiKey: process.env.MAILGUN_API_KEY,
+  domain: process.env.MAILGUN_DOMAIN,
+});
 
 const mgModule = {};
 
-mgModule.sendPasswordResetEmail = (receiver, token) => {
-  const link = `https://learndatabases.dev/setPassword/${token}`;
+mgModule.sendPasswordResetEmail = (receiver, token, hostname = "https://learndatabases.dev") => {
+  const link = `${hostname}/setPassword/${token}`
```

Comment:
instead of argument, why not use `process.env` directly? This way you don't have to change the function shape. On line 6 and 7 the `env` function is used already.